### DISCUSSION TITLE
Połączenie ActiveDirectory - Azure AD Connect czy Azure AD Connect w chmurze  - AzureActiveDirectory

### DISCUSSION THREAD

**Discussion by sjarcz** [AzureCommunityPL/group#335] [Pomoc]
Witam,

poszukuję pomocy przy synchronizacji AD lokalnego z AD w Azure.

Sytuacja wyglada następująca.

W AD w Azure jest domena oprócz nazwafirmy.onmicrosoft.com również domena nazwafirmy.pl i  już spora ilość kont (ponad 200) użytkowników z loginami i adresami e-mail odpowiednio:
jnowak@nazwafirmy.pl
akowalski@nazwafirmy.pl
ekwiatkowska@nazwafirmy.pl
......

W lokalnym AD mam użytkowników w domenie lokalnej nazwafirmy.local
jnowak@nazwafirmy.local
akowalski@nazwafirmy.local
ekwiatkowska@nazwafirmy.local
......

Czy jest możliwa taka synchronizacja aby konta z AD lokalnego połączyły się w jedno z odpowiednimi kontami AD w Azure i w efekcie przy pomocy jednego poświadczenia można było się logować na komputer, do domeny lokalnej oraz office365 i innych usług? 

Jak narazie przy różnych próbach udaje mi się otrzymać nowe konta w AD Azure po synchronizacji i w efekcie są konta:
jnowak@nazwafirmy.pl
i
jnowak@nazwafirmy.onmicrosoft.com

Czy przy tej synchronizacji powinienem korzystać z Azure AD Connecta czy Azure AD Connecta w chmurze.

Nie wiem czy nie za bardzo chaotycznie ale może ktoś coś doradzi a jak trzeba to uściślę jak są niejasności.

Dziękuję za wszelką pomoc i sugestie

**Comment by krukowskid** [comment]
dodaj do AD suffix nazwafirmy.pl i ustaw go wszystkim jako główny (oczywiście wcześniej ich informując). To ujednolici logowanie chmura/on-prem,  bo wtedy upn = email. - nie jest to obowiązkowe 

łączenie kont - jeżeli nie były one wcześniej synchronizowane tylko zostały utworzone w AAD to ustaw parametr łączenia w AD connect na email i upewnij się, że użytkownik w AD ma wypełnione pole email i jest ono spójne z chmurowym loginem. 
Dla większej ilości informacji poszukaj pod hasłami "soft matching" i "hard matching"
